DorisProject/src/app/pages/modal/modal.component.ts

```diff
@@ -0,0 +1,40 @@
+import {Component, Inject, OnInit} from '@angular/core';
+import {MAT_DIALOG_DATA, MatDialogRef} from '@angular/material/dialog';
+import {Router} from '@angular/router';
+
+@Component({
+  selector: 'app-modal',
+  templateUrl: './modal.component.html',
+  styleUrls: ['./modal.component.scss']
```

Comment:
Codacy has a fix for the issue: [Insert `,`](https://app.codacy.com/gh/SaraArenasV/DorisProjectFrontEnd/pullRequest?prid=9258681)
```suggestion
  styleUrls: ['./modal.component.scss'],
```